ui/src/app/pages/profile/data-user-code-of-conduct.tsx

```diff
@@ -231,6 +232,7 @@ export const DataUserCodeOfConduct = withUserProfile()(
                       data-test-id={'submit-ducc-button'}
                       disabled={errors || submitting}
                       onClick={() => {
+                        AnalyticsTracker.Registration.AcceptDUCC();
```

Comment:
You can view this page from the profile as well. Does it make sense this bump this up to a higher level callback in the registration flow instead? Since the metric is registration specific

Reply:
That does make sense, but doesn't look like there's a higher callback. The submit function just calls `window.history.back()` when complete.

I'm not too familiar with this workflow but seems like we should disable the Accept button if the user has already accepted the agreement. Currently they can view and accept as many times as they want.

Reply:
There are no current plans to change this. Before I changed the workflow you could still re-sign the DUCC from the profile page. I'm not sure whether we still want to do that, that's really more of a product question. I could see either allowing or disallowing re-signing making sense.